package.json

```diff
@@ -301,6 +301,12 @@
           "type": ["integer", "null"],
           "default": null
         },
+        "ruff.prioritizeFileConfiguration": {
+          "markdownDescription": "Whether to prioritize file configuration over configuration set in extension settings.",
```

Comment:
I don't think I would understand what that means. We may need to give an example here.